sqlserver/assets/configuration/spec.yaml

```diff
@@ -129,12 +129,225 @@ files:
         type: boolean
         example: false
         display_default: true
-    - name: include_ao_metrics
+    - name: database_metrics
```

Comment:
Wdyt about making all of these metrics `hidden: true` so they don't pollute the `conf.yaml.example` which customers use to view all configuration options. I think for most customers, all of these create an overwhelming number of knobs.

Reply:
This is something Zhengda and I discussed, I'm curious what @christinaduffy57 considers best/cleanest here. Can follow up on slack.